src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.c

```diff
@@ -358,7 +357,7 @@ static grpc_ares_request *grpc_dns_lookup_ares_impl(
     grpc_ares_request_ref(r);
     char *service_name;
     gpr_asprintf(&service_name, "_grpclb._tcp.%s", host);
-    ares_query(*channel, service_name, ns_c_in, ns_t_srv, on_srv_query_done_cb,
+    ares_query(*channel, service_name, 1 /*ns_c_in*/, 33 /*ns_t_srv*/, on_srv_query_done_cb,
```

Comment:
@ax3l Thanks for the explanation!

@markdroth Sounds great. Let's check in this PR first? I'll send a follow-up PR to add the constants in port_platform.h.